### PR TITLE
version 0.0.80; bump to unstructured 0.15.10

### PR DESCRIPTION
**Pull request by MthwRobinson**
### Summary

Bumps to `unstructured==0.15.10`.

### CODE REVIEW

**Comment by MthwRobinson** [comment]
Dockerfile test will be fixed once this is merged upstream in `base-images`

- https://github.com/Unstructured-IO/base-images/pull/44

**Review by christinestraub — APPROVED** [review]

File: requirements/base.in [hunk 1]
```diff
@@ -1,5 +1,6 @@
 -c constraints.in
 unstructured[all-docs]>=0.8.1
+unstructured-client==0.25.5
```
Comment (Coniferish) [inline requirements/base.in:3]: Overall LGTM. Why did this need to be pinned though?

Comment (MthwRobinson) [reply requirements/base.in:3]: Investigating a failing test
- https://github.com/Unstructured-IO/unstructured-api/actions/runs/10793859754/job/29947819866?pr=458

File: test_general/api/test_app.py [hunk 2]
```diff
@@ -601,7 +601,7 @@ def test_parallel_mode_preserves_uniqueness_of_hashes_when_assembling_pages_spli
     response = client.post(
         MAIN_API_ROUTE,
         files=[("files", (str(test_file), open(test_file, "rb"), "application/pdf"))],
-        data={},
+        data={"chunking_strategy": "by_title", "new_after_n_chars": 500},
```
Comment (MthwRobinson) [inline test_general/api/test_app.py:604]: @awalker4 - I get `<Response [422 Unprocessable Entity]>.status_code` when I use `data={}`, and the associated error message is below. When you explicitly set the params it complains about, the test works fine. Will check and see if any dependency bumps seem like they'd cause this.

```
ERROR    unstructured_api:app.py:44 [{'type': 'literal_error', 'loc': ['body', 'chunking_strategy'], 'msg': "Input should be 'by_title'", 'input': '', 'ctx': {'expected': "'by_title'"}}, {'type': 'int_parsing', 'lo
c': ['body', 'new_after_n_chars'], 'msg': 'Input should be a valid integer, unable to parse string as an integer', 'input': ''}]
```

Comment (MthwRobinson) [reply test_general/api/test_app.py:604]: Nothing jumps out unless it was the `fastapi` version bump. I'll give that a try  quick to test.

Comment (MthwRobinson) [reply test_general/api/test_app.py]: Confirmed, the `fastapi` bump is what caused the failure. Works with `fastapi==0.112.x` and `fastapi==0.113.x` but not `fastapi==0.114.x`.

Comment (MthwRobinson) [reply test_general/api/test_app.py]: Pinning to `fastapi==0.113.0` for number to preserve behavior, but we should circle back on how to safely upgrade `fastapi`.

**Comment by jsoref** [comment]
When this merged, CI started failing ... https://github.com/Unstructured-IO/unstructured-api/actions/runs/10799719598

**Pull request merged**